<?xml version="1.0.0" ?>

<!-- 
  This launch file runs the object tracker
-->  
<launch>

  <!-- Start the node in the AGV namespace -->
  <group ns="$(optenv NAMESPACE default)">

    <!--
    Run object tracker node
    -->
    <node pkg="object_tracker" type="object_tracker_node" name="object_tracker" output="screen" required="true">
      <param name="ra_motor_step_per_revolution" value="200" />
      <param name="dec_motor_step_per_revolution" value="200" />
      <param name="ra_motor_microstep" value="16" />
      <param name="dec_motor_microstep" value="16" />
      <param name="ra_gear_ratio" value="3" />
      <param name="dec_gear_ratio" value="3" />
      <param name="ra_worm_gear_ratio" value="130" />
      <param name="dec_worm_gear_ratio" value="65" />
    </node>

  </group>

</launch>

